cobalt/base/wrap_main.h

```diff
@@ -68,68 +68,6 @@ void NoopStopFunction() {}
 
 #if defined(STARBOARD)
 #include "cobalt/base/wrap_main_starboard.h"
-#else
```

Comment:
Could you check if these are used by the native or host toolchains in trunk? I.e. if you make this same change (just for this file) in trunk, I assume we'd expect it to pass

Reply:
> Could you check if these are used by the native or host toolchains in trunk? I.e. if you make this same change (just for this file) in trunk, I assume we'd expect it to pass

I think it's unlikely to still be used but it's a good idea to give that a try and see of all the builds are green.

Reply:
Running at #2543

Reply:
It's all green. I guess that PR can just be merged. It shouldn't conflict with the same change here when merging back.

Reply:
After adding on_device, there are some failures, I'll investigate.